c/extras/aggregator.cc

```diff
@@ -0,0 +1,337 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
license header truncated

Reply:
Yes, something happened to my editor when pasted. Fixed.

Reply:
Not sure where to comment on the review in general. So will do it here. Thanks a lot for your time, 

1) Yes, this is something I've also noticed when looked at the code on `github`. It looks ok in the editor, though. I will find the way to fix it.

2) I've already merged the `master` branch into the `aggregator` and found the problems you mention. Fixed the code so that it can be built.

3) Yes, getting rid out of the warnings now.

4) Sure, that's an easy thing to do.

Thanks!